metapool/count.py

```diff
@@ -160,6 +169,25 @@ def bcl2fastq_counts(run_dir, sample_sheet):
     return out
 
 
+def _bclconvert_counts(path, sample_sheet):
+    # read the csv in from file
+    df = pd.read_csv(path)
+    # subselect only the columns we're concerned with
+    df = df[["SampleID", "Lane", "# Reads"]]
+    # filter out rows that reference and 'Undetermined' fastq.gz file
```

Comment:
```suggestion
    # filter out rows that reference an 'Undetermined' fastq.gz file
```

Reply:
@ElDeveloper I'll add an additional test to seqpro, thanks for the suggestion!

I was thinking it would be good to raise an Error on both as well, but I'm not sure if realistically both would ever be in the same directory, since the fastq files are either going to be generated by bcl2fastq or bcl-convert, but not both. What do you think? Right now if it sees both, it will take Stats directory over what bcl-convert generates.